client/blocks/comment-detail/docs/comment-faker.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { keyBy, omit } from 'lodash';
+
+/**
+ * `CommentFaker` is a HOC to easily test the Comments Management without the necessity of real data or actions.
+ *
+ * It takes the comments passed via props (with mock data, Redux-connected, etc.) and puts them in its internal state.
+ * Then passes them and various actions (status and like setters) to the actual comment list component.
+ *
+ * Once data and actions will be completely Reduxified, it will be enough to remove the `CommentFaker` call
+ * and use the `mapDispatchToProps` actions instead, leaving this component still useful for testing purposes.
+ */
+export const CommentFaker = WrappedCommentList => class extends Component {
+	state = {
+		comments: {},
+	};
+
+	componentWillMount() {
+		if ( this.props.comments.length ) {
+			this.setState( { comments: keyBy( this.props.comments, 'ID' ) } );
+		}
+	}
```

Comment:
oh, so you think it could send props with 6 comments, then 0 comments, then 8 comments or something like that and you want to make sure we don't wipe out the comments?